Error: File 'UI/flashcards.ui' is not valid

while executing '/usr/local/lib/python3.10/dist-packages/PySide2/uic -g python UI/flashcards.ui'
